content/md/retailer-onboarder/first-steps/terms-and-conditions.md

```diff
@@ -0,0 +1,35 @@
+---
+id: terms-and-conditions
+themes: 
+title: Terms and conditions
+popular: false
+---
```

Comment:
```suggestion
id: ethnio
themes: 
title: Ethn.io Privacy Policy
popular: false
---
```

We should be more explicit into filename + it will help us to add a link into ethnio screeners (shorter one)